chroma_core/models/stratagem.py

```diff
@@ -325,10 +327,12 @@ def calc_warn_duration(report_duration, purge_duration):
 
             return "&& != type S_IFDIR < atime - sys_time {}".format(report_duration or 0)
 
-        def get_body(mount_point, report_duration, purge_duration):
+        def get_body(mount_point, action, report_duration, purge_duration, search_expression):
```

Comment:
So cloudsync and filesync can't be run simultaneously?

Reply:
Correct.  There is currently no requirement for that.